delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py

```diff
@@ -52,8 +52,8 @@ def test_delivery_free_fee_removal_with_fee(self):
             [
                 {
                     "is_free_delivery": False,
-                    "qty_to_invoice": 1,
-                    "invoice_status": "to invoice",
+                    "qty_to_invoice": 0.0,
```

Comment:
Why these changes in tests ?

Reply:
Odoo14 prevent trigger sale_order.product_id.invoice_status after confirm, Sale order not yet delivered so nothing to invoice, I add more test case for this module in PR